<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GGQ6OF8" failures="1" tests="2" name="Test" time="6.938" errors="0" timestamp="2024-08-23T22:04:44 IST">
  <testcase classname="com.SwagLabs.Testlayer.LoginTest" name="validatelogintest" time="0.26"/>
  <testcase classname="com.SwagLabs.Testlayer.LoginTest" name="validatelogintestwithinvalidcredntials" time="0.215">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.SwagLabs.Testlayer.LoginTest.validatelogintestwithinvalidcredntials(LoginTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validatelogintestwithinvalidcredntials -->
</testsuite> <!-- Test -->
